### DISCUSSION TITLE
Fireplace Not Working

### DISCUSSION THREAD

**Discussion by HurtaBaby** [aarons22/homebridge-bond#232] [Q&A]
I only have one device on my bond, my fireplace. Everything works in the bond app (on, off, max flame, min low flame, flame up, flame down) but it is not working at all in home app. I see one toggle switch for on/off and one with a light bulb thinking that it would change the flame height. However, every time I toggle the switch nothing happens.  Only thin that works is slight the light dimmer to high to turn on and the dimmer all the way down to turn off. Any help would be much appreciated. Thanks!

**Comment by TeslaOwnerTips** [comment]
Check issues
#230
#225
#188
It appears that not much is being done with the fireplace. Maybe because the developer(s) don't have one.
